### DISCUSSION TITLE
DNS 经过代理查询问题

### DISCUSSION THREAD

**Discussion by malagebidi** [MetaCubeX/mihomo#819] [Q&A]
Clash.Meta的dns配置部分，如下：
  nameserver:
    - https://dns.google/dns-query#$PROXY
    - https://security.cloudflare-dns.com/dns-query#$PROXY
    - quic://dns.adguard-dns.com
  nameserver-policy:
    "geosite:cn":
      - https://223.5.5.5/dns-query#h3=true

根据文档，此时geosite非cn的域名，将使用nameserver中配置的doh交由PROXY进行远程解析。我的节点使用的是vless，里面也配置了dns的部分如下：
  "servers": [
    "https+local://1.1.1.1/dns-query",
    "localhost"
  ],
  "tag": "dns_inbound"

我要问的问题是：远程解析是由哪个dns完成的？是Clash.Meta配置的doh还是我节点中配置的dohl，或者说重复进行了解析。

**Comment by Rafeal-Ma** [comment]
应该是先确定DNS模式是fakeip还是redir-host，如果是FAKEIP,那么你非直连的域名都交给了你节点的服务器来解析。GEOSITE:CN以外的直连 才会交给你配置的nameserver来解析。
